requirements/local.txt

```diff
@@ -1,17 +1,60 @@
-# Local development dependencies go here
--r base.txt
+appdirs==1.4.3
+awesome-slugify==1.6.5
+boto==2.43.0
+certifi==2017.4.17
+chardet==3.0.3
+configparser==3.5.0
 coverage==4.2
-django-coverage-plugin==1.3.1
-Sphinx==1.5
-django-extensions==1.7.5
-Werkzeug==0.11.11
+defusedxml==0.5.0
+Django==1.10.4
+django-allauth==0.29.0
+django-anymail==0.6.1
+django-braces==1.10.0
+django-crispy-forms==1.6.1
+django-debug-toolbar==1.8
+django-environ==0.4.1
+django-extensions==1.7.9
+django-model-utils==2.6
+django-redis==4.6.0
+django-storages-redux==1.3.2
 django-test-plus==1.0.16
-factory_boy==2.8.1
-
-django-debug-toolbar==1.6
-
-# improved REPL
-ipdb==0.10.1
-
+djangorestframework==3.6.2
+enum34==1.1.6
+environ==1.0
+factory-boy==2.7.0
+fake-factory==0.7.4
+flake8==3.2.1
+funcsigs==1.0.2
+gevent==1.2a1
+greenlet==0.4.12
+gunicorn==19.6.0
+idna==2.5
+ipaddress==1.0.18
+mccabe==0.5.3
+mock==2.0.0
+oauthlib==2.0.2
+packaging==16.8
+pbr==3.0.0
+Pillow==3.4.2
```

Comment:
You can see that you ended up putting more libraries than what the project requires. If this was a team repo, other developers would run into problem because of the extra dependencies you added that isn't being used by the project.